components/calculator.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Buttons from './buttons';
+import Display from './display';
+import calculate from '../logic/calculate';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
```

Comment:
- [ ] This should be a functional-based component as requested by the previous reviewer. Kindly update this component.
- [ ] Also note that Linters are great tools that make sure your ways of coding are up to standard. Kindly fix the code issues and do not disable them